README/ReleaseNotes/v624/index.md

```diff
@@ -79,6 +79,21 @@ See the discussion at [ROOT-11014](https://sft.its.cern.ch/jira/browse/ROOT-1101
 - Just-in-time compilation of string expressions passed to `Filter` and `Define` now generates functions that take fundamental types by const value (rather than by non-const reference as before). This will break code that was assigning to column values in string expressions: this is an intended side effect as we want to prevent non-expert users from performing assignments (=) rather than comparisons (==). Expert users can resort to compiled callables if they absolutely have to assign to column values (not recommended). See [ROOT-11009](https://sft.its.cern.ch/jira/browse/ROOT-11009) for further discussion.
 - RDataFrame action results are now automatically mergeable thanks to the new interface provided by `ROOT::Detail::RDF::RMergeableValue` and derived, introduced in [#5552](https://github.com/root-project/root/pull/5552). A feature originally requested with [ROOT-9869](https://sft.its.cern.ch/jira/browse/ROOT-9869), it helps streamline RDataFrame workflow in a distributed environment. Currently only a subset of RDataFrame actions have their corresponding mergeable class, but in the future it will be possible to extend it to any action through the creation of a new `RMergeableValue` derived class.
 
+### Distributed computing with RDataFrame
+ROOT 6.24 introduces `ROOT.RDF.Experimental.Distributed`, an experimental python package that enhances RDataFrame with distributed computing capabilities. Users will be able to distribute their RDataFrame applications with the supported backend of their choice, while maintaining almost exactly the same code. The backend submodules of this package will expose their own `RDataFrame` objects. The only needed change in user code is to substitute `ROOT.RDataFrame` calls with such backend-specific `RDataFrame`s. Currently the [Apache Spark](http://spark.apache.org/) backend is supported, an example usage of this backend would look like:
```

Comment:
To be symmetrical with other C++ classes, `ROOT.Experimental.RDF.Distributed` might be less surprising. Is `ROOT.RDF` a thing already?

Reply:
Yes ROOT.RDF is already used to call for example ROOT.RDF.MakeCsvDataFrame . We decided that the final name for this package would be ROOT.RDF.Distributed so I thought to put the "Experimental" keyword after the name of the cpp namespace/python module that will host this feature. If you think it should go one step above we can still change it